.storybook/preview.js

```diff
@@ -1,49 +1,56 @@
+import React from 'react';
 import { addParameters } from "@storybook/react";
 import FDS from 'lib/dictionary/js/styleConstants';
+import {
+  Title,
+  Subtitle,
+  Description,
+  Primary,
+  ArgsTable,
+  Stories,
+  PRIMARY_STORY,
+} from '@storybook/addon-docs/blocks';
+import ImportPath2 from 'util/storybook-docs/ImportPath2';
 
 import "../src/components/style/index.css";
 
-addParameters({
-  backgrounds: [
-    { name: "White", value: FDS.COLOR_WHITE },
-    { name: "Haze", value: FDS.COLOR_HAZE},
-    { name: "Navy", value: FDS.COLOR_NAVY }
-  ],
+export const parameters = {
+  docs: {
+    page: () => (
+      <>
+        <Title />
+        <ImportPath2 />
```

Comment:
This is the only additional thing we're adding to the doc page, an import statement right below the title.